cmd/celestia/main.go

```diff
@@ -35,6 +38,13 @@ func main() {
 }
 
 func run() error {
+	seed := make([]byte, 32)
+	_, err := crypto_rand.Read(seed)
+	if err != nil {
+		return err
+	}
+	math_rand.Seed(int64(binary.LittleEndian.Uint64(seed)))
```

Comment:
As per https://github.com/celestiaorg/celestia-node/pull/850/files#r906162596, lets simplify the code here by using time.Unix and we are good to go with this PR. Thanks